bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -1113,6 +1112,35 @@ fn start_trace_agent(
     )
 }
 
+/// Sends metrics indicating issue with configuration.
+///
+/// # Arguments
+/// * `issue_reasons` - Vector of messages describing the issue with the configurations
+/// * `config` - The config instance (from fallback compatibility agent)
+/// * `metrics_aggr` - The metric aggregator
+fn send_config_issue_metric(
+    issue_reasons: &[String],
+    config: &Arc<Config>,
+    metrics_aggr: &Arc<Mutex<MetricsAggregator>>,
+) {
+    if issue_reasons.is_empty() {
+        return;
+    }
+    let lambda_enhanced_metrics =
+        enhanced_metrics::new(Arc::clone(metrics_aggr), Arc::clone(config));
```

Comment:
Could we reuse this too? Create it, use it, then send it to the lifecycle processor? That way we don't allocate twice?

You won't get the right amount of tags after the invocation has occurred, like runtime